src/internal/connector/data_collections_test.go

```diff
@@ -582,15 +596,48 @@ func (suite *ConnectorCreateSharePointCollectionIntegrationSuite) TestCreateShar
 		t      = suite.T()
 		siteID = tester.M365SiteID(t)
 		gc     = loadConnector(ctx, t, Sites)
-		sel    = selectors.NewSharePointBackup()
 	)
 
-	sel.Include(sel.Libraries(
-		[]string{siteID},
-		[]string{"foo"},
-		selectors.PrefixMatch(),
-	))
+	tables := []struct {
+		name     string
+		sel      func(t *testing.T) selectors.Selector
+		expected int
+	}{
+		{
+			name:     "SharePoint.Libraries",
+			expected: 0,
+			sel: func(t *testing.T) selectors.Selector {
+				sel := selectors.NewSharePointBackup()
+				sel.Include(sel.Libraries(
+					[]string{siteID},
+					[]string{"foo"},
+					selectors.PrefixMatch(),
+				))
 
-	_, err := gc.DataCollections(ctx, sel.Selector, nil, control.Options{})
-	require.NoError(t, err)
+				return sel.Selector
+			},
+		},
+		{
+			name:     "SharePoint.Lists",
+			expected: 1,
+			sel: func(t *testing.T) selectors.Selector {
+				sel := selectors.NewSharePointBackup()
+				sel.Include(sel.Lists(
+					[]string{siteID},
+					selectors.Any(),
```

Comment:
We might want to narrow this down a bit.  Could probably be `"TODO"` for now.  In the future we'll probably want a curated site to test against, with a guaranteed default list of some kind to isolate queries against.

Reply:
I am unclear about this statement. Given the `siteID`, the query will return exactly one collection.  Within this collection, there are several (4 at last count) lists that are returned.

Reply:
Resolved within [9b3af77](https://github.com/alcionai/corso/pull/1788/commits/9b3af77c07de84819f531f1425078cd5b5a89e3b)